feature_engine/transformation/log.py

```diff
@@ -147,6 +149,45 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             X.loc[:, self.variables_] = np.log10(X.loc[:, self.variables_])
 
         return X
+    
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Transforms the variables using exp transformation.
```

Comment:
could we say exponential instead of exp? Could we also add "reverses the logarithm transformation"